### PR TITLE
[MODULAR] Christmas tree has a whitelist for presents

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Adds a 'safe' version of the xmas tree that only hands out gifts based on a determined list, instead of handing out literally anything such as debug and admin gear.

image

## How This Contributes To The Skyrat Roleplay Experience
Something Happened.

## Changelog

:cl: LT3
add: Christmas tree no longer hands out debug and admin gear.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/holidays/flora.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	icon_state = "pinepresents"
+	desc = "A wondrous decorated Christmas tree. It has presents!"
+	gift_type = /obj/item/a_gift //only give safe gifts from the tree
+	unlimited = FALSE
+
+/obj/structure/flora/tree/pine/xmas/presents/safe/unlimited
+	icon_state = "pinepresents"
+	desc = "A wondrous decorated Christmas tree. It has presents!"
+	gift_type = /obj/item/a_gift //only give safe gifts from the tree
+	unlimited = TRUE
```
Comment (Jolly-66) [inline modular_skyrat/modules/holidays/flora.dm:11]: File isn't ticked, which is causing lints to fail.

Comment (lessthnthree) [reply modular_skyrat/modules/holidays/flora.dm:11]: Oh I reverted changes to tgstation.dme, so yeah. Derp.

**Comment by dawsonkeyes** [comment]
image

**Comment by lessthnthree** [comment]
I think people have been pretty responsible, but was told it's this or no more trammas tree.

**Review by Zonespace27 — APPROVED** [review]
For the downvoters: this doesn't affect the _actual_ chrimmas tree

**Comment by Zonespace27** [comment]
however, i must do this

**Comment by softcerv** [comment]
> I think people have been pretty responsible, but was told it's this or no more trammas tree.

That sucks to hear.

**Comment by lessthnthree** [comment]
> however, i must do this

There, added photo.

**Review by GoldenAlpharex — APPROVED** [review]
I asked for this, good change, I didn't have to make it myself. 👍

**Pull request merged**